### DISCUSSION TITLE
xDrip on phone with LineageOS 18.1 and microG: recurring warning

### DISCUSSION THREAD

**Discussion by steve8x8** [NightscoutFoundation/xDrip#3175] [General]
I'd like to replace my old handset with a not-so-old Sony Z5compact. For that, I can get LineageOS 18.1, 19.1 and 20.0 with recent security fixes - but the ROM comes with microG preinstalled.
Since I don't need Google services, I'd be perfectly happy with that. Unfortunately, xDrip keeps emitting a warning "xDrip won't run without Google Play services, which are not supported by your device" that has to be confirmed periodically - which will even "pile up" so functions of the app can't be reached immediately anymore.
I presume this is due to some cloud interaction features - which I'd happily sacrifice: I'm uploading to NightScout and followers may use that.
Is there a means to silence those warnings? At least to avoid the piling-up?
A logcat (zipped) is attached...
[logcat.zip](https://github.com/NightscoutFoundation/xDrip/files/13278653/logcat.zip)
Thanks for any advice, I'd prefer not to replace microG with GApps on that device.

**Comment by Navid200** [comment]
Have you disabled sync?
https://navid200.github.io/xDrip/docs/GooglePlayServices_NotInstalled.html
